### PR TITLE
update actix to version 0.13

### PR DESCRIPTION
**Pull request by LemonHX**
Signed-off-by: LemonHX <lemonhx@lemonhx.tech>

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: cmd/miner_client/src/main.rs [hunk 1]
```diff
@@ -39,10 +39,7 @@ fn main() {
         }
     };
     let user = opts.user;
-    let mut system = System::builder()
-        .stop_on_panic(true)
-        .name("starcoin-miner")
-        .build();
+    let system = System::new();
```
Comment (LemonHX) [inline cmd/miner_client/src/main.rs:42]: wait to update tokio
and then change to `System::with_tokio_rt()`

File: commons/service-registry/tests/panic.rs [hunk 2]
```diff
@@ -15,10 +15,7 @@ pub mod common;
 #[stest::test]
 fn test_service_panic() {
     //Fixme Expect the system stop on panic,but this feature not work as expect.
-    let mut sys = System::builder()
-        .stop_on_panic(true)
-        .name("panic_test")
-        .build();
+    let sys = System::new();
```
Comment (LemonHX) [inline commons/service-registry/tests/panic.rs:18]: wait to update tokio
and then change to `System::with_tokio_rt()`

File: commons/stest/stest-macro/src/lib.rs [hunk 3]
```diff
@@ -152,17 +152,11 @@ pub fn test(args: TokenStream, item: TokenStream) -> TokenStream {
...
-
-                let local = stest::LocalSet::new();
-                let future = stest::actix_export::System::run_in_tokio("test", &local);
-                local.spawn_local(future);
-
-                stest::actix_export::Arbiter::spawn(stest::timeout_future(#timeout,tx.clone()));
-                stest::actix_export::Arbiter::spawn(stest::test_future(async{ #body },tx));
-
-                local.block_on(&mut rt,stest::wait_result(rx))
+                let system = stest::actix_export::System::new();
```
Comment (LemonHX) [inline commons/stest/stest-macro/src/lib.rs:155]: @jolestar IDK this is correct behavior or not

File: rpc/client/src/lib.rs [hunk 4]
```diff
@@ -135,7 +135,8 @@ impl RpcClient {
         let inner: RpcClientInner = provider.get_rpc_channel().map_err(map_err)?.into(); //Self::create_client_inner(conn_source.clone()).map_err(map_err)?;
         let pubsub_client = inner.pubsub_client.clone();
         let handle = std::thread::spawn(move || {
-            let sys = System::new("client-actix-system");
+            // client-actix-system
```
Comment (LemonHX) [inline rpc/client/src/lib.rs:138]: wait to update tokio
and then change to `System::with_tokio_rt()`

**Review by LemonHX — COMMENTED** [review]
self review

**Review by jolestar — APPROVED** [review]

**Review by sanlee42 — APPROVED** [review]

**Pull request merged**